cli/src/config.rs

```diff
@@ -1528,27 +1582,15 @@ mod tests {
         let tmp = setup_config_fs(case.files);
         let env = resolve_config_env(&case.env, tmp.path());
 
-        let expected_existing: Vec<PathBuf> = case
-            .wants
-            .iter()
-            .filter_map(|want| match want {
-                Want::Existing(path) => Some(tmp.path().join(path)),
-                _ => None,
-            })
-            .collect();
-        assert_eq!(
-            env.existing_user_config_paths().collect_vec(),
```

Comment:
Test for `existing_user_config_paths()` appears deleted by mistake. Can you restore?

Reply:
It wasn't really a mistake, but yeah I can do that.